app/src/main/java/com/example/tour_a_versity_v3/dto/LocationDTO.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.tour_a_versity_v3.dto
+
+import com.google.android.gms.maps.model.LatLng
+
+data class LocationDTO(var lat: Double, var lng: Double, var title: String, var snippet: String) {
```

Comment:
I created a sample DTO class that you can either use directly, or modify to fit your needs. Currently this class is not being used anywhere in your code, it just offers an example of how you can abstract the details of the data handling away from your activity and ViewModels.

Reply:
Since this is not used anywhere, I'm cautious about merging it into master.  Only fully complete features, which have met the team's Definition of Done, should be merged to master.